src/controller/httpsAPI.ts

```diff
@@ -164,15 +170,17 @@ export class HttpsAPI {
       '/address/:address',
       async (req, res, next: NextFunction) => {
         try {
-          const { limit, prev, next, chainId = '31', blockNumber = '0' } = req.query
+          const { limit, prev, next, chainId = '31', blockNumber = '0', startdate = '0', enddate } = req.query
```

Comment:
Copy-pasted comment:

This should be startDate and endDate (camel case)

Also be sure that these parameters convey their purpose. If the expected dates are unix timestamp, then rename it to unixStartDate and unixEndDate

Reply:
same as above